src/utils/column-helper.ts

```diff
@@ -26,9 +26,13 @@ export function setColumnDefaults(columns: TableColumn[]) {
     }
 
     // format props if no name passed
-    if(column.prop != null && !column.name) {
+    if(column.prop != null && column.name == null) {
```

Comment:
Can you convert these to `!==` and `===`

Reply:
I can but it will take up twice as much code to check for null and undefined. Is that what you want?

Reply:
We do this strict checking everywhere else, i'd like to keep it consistent unless that type of checking is needed.

Reply:
Makes sense. I saw it wasn't using it here so I kept it that way with my changes. In fact, my last PR didn't do strict checking as well, for the other IE/Edge issue. I will do strict checking for changes going forward.